### DISCUSSION TITLE
Assign single analog to key/controller mapping

### DISCUSSION THREAD

**Discussion by Ugyuu** [AntiMicroX/antimicrox#912] [Q&A]
Heya! Here's an odd request, but I will try my best to describe the issue.

I'm using Donkey Konga bongos for certain games and a simulator similar to "Clone Hero" and would like to get said bongo controller working with AntiMicroX on Linux.

This controller uses 5 buttons. There is also a microphone for the "clap" inputs, which it processes as an analog signal.
On Windows, I can get clapping to work in the game by using JoyToKey, then assigning the analog input to a key on the keyboard and tweaking the threshold so it does not trigger at the lightest breeze. I can then use it in the simulator.
Steam controller setup recognizes the analog clap input, but I cannot launch the simulator through Steam without crashing.

Sadly, this is the issue I'm running into: I am unable to assign said analog input in AntiMicroX so that it can be bound to a key on the keyboard.

The analog input is recognized by the system as **Axis 5.** Its value ranges between -5462 (no noise) and 0 (max) 
_(Above might be inaccurate as I have attempted to calibrate it)_

In AntiMicroX, the analog input of axis 5 is recognized and read within the Controller Mapping screen, with the value starting at -30198 and the highest I could get being around -2185.
I cannot seem to assign said axis to any of the controller mappings no matter how hard I try, and none of the axis detection dead zone settings seem to help with this. I even set it to its lowest -- 5000, but no effect. 
I also can't use it anywhere when making button assignments either. It's there, but unusable. **All other buttons work.**
Could it be that Axis 5 can't be assigned to anything because it is a negative value?

In short, I would like to know if it is possible to assign an analog input to a button on the virtual "controller"/keyboard, so that I can then try to use it for input. Many thanks in advance for your time!
